gengen/main.go

```diff
@@ -69,6 +69,7 @@ func main() {
 	var defaultSeed = time.Now().Unix()
 
 	jsonout := flag.Bool("json", false, "sets output to be json")
+	useSmallSectors := flag.Bool("use-small-sectors", false, "use small sectors instead of full size ones")
```

Comment:
SUGGESTION: In a very short time, `use-small-sectors` will need to change to `use-test-sectors`. There will be several sector sizes, so "small" won't be particularly useful in disambiguating.